variables.tf

```diff
@@ -47,3 +47,32 @@ variable "requires_session" {
   description = "A value that indicates whether the queue supports the concept of sessions"
   default     = false
 }
+
+variable "create_managed_identity" {
+  default = false
+}
```

Comment:
I don't think this is needed here, everyone generally has created as part of key vault module, probably don't want to add that in multiple places?

Reply:
Yeah I wasn't sure. I have removed the create bits.